download.py

```diff
@@ -778,13 +799,15 @@ def download_post(task: daemon_tasks.Task):
         or ((at == "brush") and wm.get("appendable") == True)
         or at == "scene"
         or at == "hdr"
+        or at == "nodegroup"
```

Comment:
This condition could be an assignment in a mathematical olympiad for the 2nd year of high school.

Every statement between ORs could be just a separate IF with return.
```
if not ((bpy.context.mode == "OBJECT" and (at == "model" or at == "material")):
    return done
if ((at == "brush") and wm.get("appendable") == True):
    return done
if at == "scene":
    return done
if at == "hdr":
    return done
if at == "nodegroup":
    return done
```

Reply:
Haha, it's really tough, seems you turned the conditions to their opposites, and you solution would ruin the code. This is a separate issue but I fixed it and now it's just 2 conditions that return.